maxim/test_runs/test_run_builder.py

```
@@ -693,12 +746,19 @@ def process_dataset_entry(
             ],
         ) -> None:
             try:
-                rowData: LocalData = row.to_dict()["data"]
-                if rowData is None:
+                row_data: LocalData = row.to_dict()["data"]
+                if row_data is None:
                     raise ValueError(f"Dataset entry {index} is missing")
```

Ruff (0.11.9)

Abstract `raise` to an inner function

(TRY301)